src/StaticWebAssetsSdk/Targets/Microsoft.NET.Sdk.StaticWebAssets.StaticWebAssets.targets

```diff
@@ -1,6 +1,6 @@
 <!--
 ***********************************************************************************************
-Microsoft.NET.Sdk.Razor.StaticWebAssets.targets
+Microsoft.NET.Sdk.StaticWebAssets.StaticWebAssets.targets
```

Comment:
Any objections if we just change this to `Microsoft.NET.Sdk.StaticWebAssets.targets` (and likewise for similarly-named files)? It does feel a bit weird to have `StaticWebAssets` in there twice.

Reply:
Yeah, it's fine if we remove the second `StaticWebAssets` bit.